finalPrep/README.md

```diff
@@ -0,0 +1,96 @@
+# Mod 1 Final Prep
+Congrats on making it to the Mod 0 Final Prep! Complete the final exercises below and be sure to carefully follow the Pull Request steps to submit your work!
+
+### Final Technical Prep
+You've learned a ton about some of the core foundations of Javascript! Show us how far you've come by completing the following exercises! You will be using your work from these exercises in your first day of Mod 1! 
+
+- [ ] Complete the [Mod Zero Hero Challenge](./modZeroHero.js)
+- [ ] Complete the [Annotation Challenge](./annotations.js) 
+
+### Refactor Previous Work
+You've learned A LOT over the last few weeks as it relates to technical content - chances are, you probably have some code from your previous exercises that is either sloppy, incorrect, poorly named, etc. Before starting Mod 1, we want you to `refactor` your code - which is the process of adjusting or improving your code for readability and accuracy. 
+
+Some things to consider as you refactor include...
+- Are my variable names easy to understand/convey the data type they are assigned to?
+- Do I have proper indentation on my functions, if/else statements, etc. ?
+- Do I follow proper conventions for capitalization, punctuation, casing, etc. as it relates to my code?
+- Are there functions that I wrote that I can make more concise while maintaining same functionality?
+
+Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
+
+- [ ] I have refactored my `section1` exercises to the best of my ability
+- [ ] I have refactored my `section2` exercises to the best of my ability
+- [ ] I have refactored my `section3` exercises to the best of my ability
+- [ ] I have refactored my `section4` exercises to the best of my ability
+
+### Time Management Prep
+In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
+- [ ] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [ ] Study/Project work time
+- [ ] Health + Wellness
+
+When you are finished, add screenshots of your calendar so we can provide feedback if needed!
+
+- `Add Week 1 Screenshot Here`[1st week](Screen Shot 2021-04-24 at 2.18.01 PM![image](https://user-images.githubusercontent.com/69317378/115972013-3a84c300-a509-11eb-8b3f-893aa44844bc.png))
+
+
+- `Add Week 2 Screenshot Here`[2nd week](Screen Shot 2021-04-24 at 2.18.25 PM![image](https://user-images.githubusercontent.com/69317378/115972002-293bb680-a509-11eb-8e5e-8f3955d36b8a.png)
+
+- `Add Week 3 Screenshot Here`[3rd week](Screen Shot 2021-04-24 at 2.19.01 PM![image](https://user-images.githubusercontent.com/69317378/115972025-4a040c00-a509-11eb-8d71-3eb324694117.png)
```

Comment:
🚨Hmm... these calendars do not account for:
- Class time
- Homework/Group work time
- Self Care